src/main/java/edu/uci/ics/crawler4j/fetcher/PageFetcher.java

```diff
@@ -88,6 +88,7 @@ public PageFetcher(CrawlConfig config) {
                                                    .setExpectContinueEnabled(false)
                                                    .setCookieSpec(config.getCookiePolicy())
                                                    .setRedirectsEnabled(false)
+                                                   .setMaxRedirects(config.getMaxRedirects())
```

Comment:
Instead of adding these one by one CrawlConfig should allow providing a custom RequestConfig